packages/next/client/experimental-script.tsx

```diff
@@ -106,7 +129,11 @@ export default function Script(props: Props): JSX.Element | null {
         requestIdleCallback(() => loadScript(props))
       })
     }
-  }, [strategy, props])
+  }, [props, strategy])
+
+  if (!process.env.__NEXT_SCRIPT_LOADER) {
+    return null
+  }
```

Comment:
Hmm, the entire import of this file is supposed to be blocked or stubbed out in the webpack configuration. Seems odd to have this here.

What happened to the `next/experimental-script` block in `webpack-config.ts`?

Reply:
Tim removed here - https://github.com/vercel/next.js/pull/19937

Following the discussion, I think it was causing issues in webpack 5